geotrek/api/v2/serializers.py

```diff
@@ -911,8 +911,9 @@ class SensitiveAreaSerializer(DynamicFieldsMixin, TimeStampedSerializer):
         structure = serializers.CharField(source='structure.name')
         published = serializers.BooleanField()
         geometry = geo_serializers.GeometryField(read_only=True, source="geom_transformed", precision=7)
-        species_id = serializers.SerializerMethodField()
-        kml_url = serializers.SerializerMethodField()
+        species_id = serializers.SerializerMethodField(read_only=True)
```

Comment:
no need to add read_only=True on SerializerMehodField, they are already read_only